src/components/Widget.tsx

```diff
@@ -97,6 +98,7 @@ export interface WidgetProps extends TransactionEventHandlers {
   tokenList?: string | TokenInfo[]
   width?: string | number
   dialog?: HTMLDivElement | null
+  disableBranding?: boolean
```

Comment:
Remove the explicit `disableBranding` prop, as it's already included in `WidgetProps` through `extends BrandingSettings`.